Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "image"
-version = "0.25.9"
+version = "0.26.0-alpha.1"
 edition = "2021"
 resolver = "2"
 
```

Comment:
Please also set `publish = false` so I don't accidentally trigger a publish of this alpha version...

Reply:
Thanks. I'm mostly paranoid about running `cargo publish` from the wrong terminal window and publishing some messy in-progress PR when I thought I was publishing a completely different crate. Making sure that the Cargo.toml will block the publish either because the version has already been used or `publish = false` is set provides an extra safeguard